op-test

```diff
@@ -713,6 +714,20 @@ class FspOpalSuite():
         return self.s
 
 
+class GcovSuite():
+    '''Simple Suite to run for Gcov, data coverage tool on distro kernel'''
+    def __init__(self):
+        self.s = unittest.TestSuite()
+        self.s.addTest(gcov_setup.GcovBuild())
+        self.s.addTest(Lcov.LcovSetup())
+        self.s.addTest(Lcov.LcovCounterZero())
+        self.s.addTest(OpTestExample.host_full_suite())
```

Comment:
it should be host test case as `self.s.addTest(RunHostTest.RunHostTest())`

Reply:
done